src/hooks/useTxNotifications.ts

```diff
@@ -23,7 +25,10 @@ const TxNotifications = {
   [TxEvent.PROCESSING]: 'Your transaction is being processed.',
   [TxEvent.PROCESSING_MODULE]:
     'Your transaction has been submitted and will appear in the interface only after it has been successfully processed and indexed.',
-  [TxEvent.AWAITING_ON_CHAIN_SIGNATURE]: 'An on-chain signature request was submitted.',
+  [TxEvent.ONCHAIN_SIGNATURE_REQUESTED]:
+    'An on-chain signature is required. Please confirm the transaction in your wallet.',
+  [TxEvent.ONCHAIN_SIGNATURE_SUCCESS]:
+    'On-chain signature was successfully submitted. Please wait for the signature to be indexed.',
```

Comment:
As the "unchecked" SDK doesn't require wallet approval, there is not necessarily an immediate signature, e.g. it is queued in a Safe.

Reply:
I'm aware, but you would still have to wait for it to be indexed, right?
What message would you suggest?

Reply:
Only if the wallet approves it immediately. I would just remove the second sentence.

Reply:
I mean, the original Safe (not the owner) needs to index the approveHash transaction. Immediate or after a week doesn't matter.

Reply:
Sure, but the transaction may not be immediately executed.

Reply:
Updated the text as per our discussion.